internal/cmd/base/server_test.go

```diff
@@ -900,3 +907,24 @@ func TestSetupWorkerPublicAddress(t *testing.T) {
 		})
 	}
 }
+
+func TestSetupWorkerCredentialStorage(t *testing.T) {
+	ctx := context.Background()
```

Comment:
TestSetupWorkerCredentialStorage(...) into internal/daemon/worker/worker_test.go you’ll eliminate the circular dep